Sources/SwiftDocC/Semantics/Symbol/DocumentationDataVariants.swift

```diff
@@ -103,10 +103,22 @@ extension DocumentationDataVariants {
         return DocumentationDataVariants<Variant>(values: [:], defaultVariantValue: nil)
     }
     
+    private var firstVariantElement: (trait: DocumentationDataVariantsTrait, variant: Variant)? {
+        // A Dictionary's order isn't stable across program executions.
+        //
+        // These values are sorted in descending order so that elements with interface languages come before the
+        // fallback value and so that Swift comes before Objective-C.
+        //
+        // Having a stable order sufficient to make the tests stable but can still have surprising results (rdar://86580516)
+        return allValues.sorted(by: { lhs, rhs in
```

Comment:
Is there a way for us to isolate this to testing? Or cache the sorted values somehow? I'm concerned about the performance implications here but I'm not sure if it's expected for the `firstValue` to be stable or not. @franklinsch may have thoughts here.

Reply:
I think instead, we could hardcode the Swift value to be the `firstValue`, which would be an O(1) lookup. If there is no Swift value, then we can call `first`, like before, which is also O(1).

Reply:
Since the number of values is known to be very small, either one or two values, the time spent is likely mostly determined by constant factors as opposed to the order, but since this is called a lot we should keep it fast. 

I added a comment that this is returning the Swift value and updated the tests to always use a Swift value as the symbol's main language to get the right behavior.